src/controllers/neighborhood.py

```diff
@@ -28,25 +28,24 @@ def get_all_neighborhoods(city=None, state=None):
     if (state):
         filter.update({'state': [state]})
     result, code = db.get_all(Neighborhood, filter)
-    if result:
-        if code == 200:
-            neighborhoods = [get_row_dict(u) for u in result]
-            # getting statistics
-            for i in range(0, len(neighborhoods)):
-                filter = {
-                          "id_neighborhood": [
-                                neighborhoods[i]['id_neighborhood']
-                          ]
-                         }
-                ratings, status = db.get_all(Rating, filter)
-                ratings = [get_row_dict(rat) for rat in ratings]
-                if (ratings):
-                    statistics = get_neighborhood_statistics(ratings)
-                    neighborhoods[i].update(statistics)
 
-            return neighborhoods, code
-        return result, code
-    return [], 200
```

Comment:
it is necessary to return to an empty list with code 200 when there is no record in the bank and when there was no error